akka-docs/src/main/java/docs/persistence/proto/FlightAppModels.java

```diff
@@ -0,0 +1,823 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: FlightAppModels.proto
+
+package docs.persistence.proto;
```

Comment:
And this one?

Reply:
I triggered compiling protos on the entire project, seems it touched more than was really needed I guess. Will remove from this PR